Display column names before export

prykon opened the pull request.

corsacca commented:
@prykon missed this part:
"Tell the user they can select which fields are exported by changing the list fields"

corsacca commented:
@prykon 
There is not a `;` on each cell even when there are not multiple values:
![image](https://user-images.githubusercontent.com/24901539/119348675-1a295f00-bc95-11eb-9793-912251eec363.png)

prykon commented:
- Added the clarification text that tells users they can select which fields are exported by changing the list fields.
- Fixed parsing issues on exported file
